package.json

```diff
@@ -15,10 +15,10 @@
     "@atlaskit/single-select": "^4.0.3",
     "@atlaskit/spinner": "^8.0.0",
     "react-gh-corner": "^1.1.2",
-    "ts-react-toolbox": "^0.1.23"
+    "ts-react-toolbox": "soswow/ts-react-toolbox#e043fec6732a6f8a8d7f3e53197fd51ee0b952b3"
```

Comment:
Placeholder while https://github.com/zzarcon/ts-react-toolbox/pull/22 is reviewed